test/data/b.txt

```diff
@@ -0,0 +1 @@
+xyz2
```

Comment:
Add general strings to make b.txt file pass uniitests( previously it was empty so lead to failure.)